frontend/modules/measurements/measurements.sagas.ts

```diff
@@ -186,10 +203,12 @@ export function* resetMeasurementColors() {
 
 export function* setMeasureEdgeSnapping({ edgeSnapping }) {
 	try {
-		if (edgeSnapping) {
-			yield Viewer.enableEdgeSnapping();
-		} else {
-			yield Viewer.disableEdgeSnapping();
+		if (yield select(selectMeasureMode) !==  '') {
```

Comment:
In vscode it shows an error here
![image](https://user-images.githubusercontent.com/42032433/112474948-257d0e80-8d68-11eb-873a-995a8a3ab1b6.png)


It can be fixed be doing this
 ```
		if ((yield select(selectMeasureMode)) !==  '') {
```

Reply:
vscode is stricter than the linter!


This is fixed